docs/main.css

```diff
@@ -0,0 +1,173 @@
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+/* Remove text decoration for all anchor elements */
+a {
+    text-decoration: none;
+}
+
+/* Overflow for npm commands */
+pre {
+    overflow: auto;
+    background-color: #F6F8FA;
+    border-radius: 3px;
+    font-family: 'Courier New', Courier, monospace;
+    padding: 8px;
+}
+
+/* All .title contents from .main section */
+.title {
+    color: #EE7C5B;
+    font-family: "Arial Black";
+    font-size: 24px;
+    margin-top: 48px;
+}
+
+/* All .contents from .main section */
+.contents {
+    font-family: "Arial";
+    font-size: 20px;
+    line-height: 24px;
+    padding-top: 8px;
+    text-align: left;
+}
+
+
+/* Header Content */
+header {
+    background: radial-gradient(100% 50%, #E9A571 0%, #EE7C5A 82%);
+    border: 1px solid #979797;
+    padding-top: 96px;
+    text-align: center;
+}
+
+.headline {
+    color: #FFFFFF;
+    font-family: "Arial Black";
+    font-size: 72px;
+}
+
+.subheadline {
+    color: #916246;
+    font-family: "Arial Black";
+    font-size: 24px;
+}
+
+.link-container {
+    background-color: rgba(174,174,174,0.65);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-top: 96px;
+    min-width: 200px;
+    padding: 20px 0px 10px;
+}
+
+.npm-badges {
+    border-radius: 5px;
+    height: 30px;
+    width: 135px;
+    padding-right: 10px;
+}
+
+.gh-btns {
+    border-radius: 5px;
+    padding-left: 5px;
+}
+
+/* Main Content flexbox */
+.main {
+    margin: 0 auto;
+    max-width: 640px;
+    padding: 24px 24px;
+}
+
+/* Info */
+.info-section .contents:nth-of-type(1) {
+    margin-top: 32px;
+}
+
+.info-section .contents:nth-of-type(2) {
+    margin-top: 16px;
+}
+
+/* Require */
+#nodelogo {
+    height: 135px;
+    padding: 8px 0px 0px 32px;
+    width: 224px;
+}
+
+#gitlogo {
+    height: 125px;
+    padding: 8px 0px 0px 32px;
+    width: 296px;
+}
+
+.require-links {
+    font-family: "Arial";
+    font-size: 24px;
+}
+
+/* Installation */
+table {
+    border: 2px solid #979797;
+    border-collapse: collapse;
+    margin-top: 16px;
+    width: 100%;
+}
+
+td, th {
+    border: 2px solid #979797;
+    font-size: 16px;
+    padding: 8px 0px 8px 8px;
+    text-align: left;
+}
+
+th {
+    color: #EE7C58;
+}
+
+.description {
+    font-family: "Arial Black";
+}
+
+.command {
+    font-family: 'Courier New', Courier, monospace;
+}
+
+tr:nth-child(even) {
+    background-color: #D5D5D5;
+}
+
+/* Footer */

```

Comment:
The footer styling doesn't seem to match the header styles. 

Take a look. Are we using a color palette guide?